scripts/build/gen_syscalls.py

```diff
@@ -159,6 +157,20 @@
 #endif
 """
 
+
+exported_template = """
+/* Export syscalls for extensions */
+static void *no_handler = NULL;
```

Comment:
```suggestion
static void * const no_handler = NULL;
```
Should spare a bit of RAM, but most importantly, hopefully relocs :slightly_smiling_face: